### PR TITLE
Xniffer post 

### PR DESCRIPTION
**Pull request by KarinaFernandez**
Introducing to Xniffer, with installation steps and example of use.

### CODE REVIEW

File: _posts/2016-04-26-Introducing-to-Ahoy.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title:  Introducing Ahoy!
+date:   2017-04-26 14:01:56
+author: XL
+categories: Swift, iOS
+markdown: redcarpet
```
Comment (enrigalmig) [inline _posts/2016-04-26-Introducing-to-Ahoy.md]: please fix the author like ahoy and include a description :)

Comment (enrigalmig) [inline _posts/2016-04-26-Introducing-to-Ahoy.md]: forgot about this, it was fixed in the other PR.

File: _posts/2017-05-02-Introducing-to-Xniffer.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title:  Introducing Xniffer, the swift network profiler built on top of URLSession!
+date:   2017-04-26 11:20:56
+author: XL
+categories: Swift, iOS, Tutorial, network
```
Comment (enrigalmig) [inline _posts/2017-05-02-Introducing-to-Xniffer.md]: but not here please fix the author like ahoy and include a description :) hahah

File: _posts/2017-05-02-Introducing-to-Xniffer.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+markdown: redcarpet
+
+---
+
+So many times we want to debug or monitoring external services responses in our app, recognize the latency and also work with the returned data.
```
Comment (patriciapoj2017) [inline _posts/2017-05-02-Introducing-to-Xniffer.md]: Do you often want to debug or monitor external service responses in you app, recognize latency and work with the data you receive back? Same happens to us!

File: _posts/2017-05-02-Introducing-to-Xniffer.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+That is way we created [Xniffer](https://github.com/xmartlabs/Xniffer) a non-intrusive framework to intercept outgoing requests and their responses. Developed in Swift for iOS platforms, inspired by [Response detective](https://github.com/netguru/ResponseDetective).
```
Comment (patriciapoj2017) [inline _posts/2017-05-02-Introducing-to-Xniffer.md]: That’s why we created Xniffer, a non intrusive framework to intercept outgoing requests and their responses. Developed in Swift for iOS platforms, inspired by Response detective.

File: _posts/2017-05-02-Introducing-to-Xniffer.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+## Why use Xniffer?
+Xniffer not only returns the responses' result, it also has a UI component that shows you those responses in real time, which other libraries don't have. It displays the latency of HTTP messages, and you can set up your own delegate to do whatever you want with the returned data.
```
Comment (patriciapoj2017) [inline _posts/2017-05-02-Introducing-to-Xniffer.md]: it also has a UI component that shows them to you in real time, feature which other libraries don't have

(sacamos response 2 veces del mismo renglón y corregimos la última parte agregando el noun feature.

File: _posts/2017-05-02-Introducing-to-Xniffer.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+All you have to do in order to use it is really easy, just setup some configurations, choose the way that you want to see the request and that's it!
```
Comment (patriciapoj2017) [inline _posts/2017-05-02-Introducing-to-Xniffer.md]: Wanna start to use it right away? Just setup some configurations, choose how you want to see the requests and that's it!

File: _posts/2017-05-02-Introducing-to-Xniffer.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+![](https://github.com/xmartlabs/Xniffer/blob/master/Example/curl.gif)
+
+## How to use it?
+To start working with Xniffer, it needs to be added as a middleware between your URLSession and the Internet. After the initial setup, a window will be displayed on top of the UI that will show all the current requests and their latency.
```
Comment (patriciapoj2017) [inline _posts/2017-05-02-Introducing-to-Xniffer.md]: To start working with Xniffer,  you’ll need to add it as a middleware between your URLSession and the Internet. After this initial setup, a window that will show you all the current requests and their latency will be displayed on top of the UI. 

(cambio de orden de la oración)

File: _posts/2017-05-02-Introducing-to-Xniffer.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+``` Swift
+let configuration = URLSessionConfiguration.default
+configuration.httpAdditionalHeaders = SessionManager.defaultHTTPHeaders
+Xniffer.setup(with: configuration)
+let sessionManager = SessionManager(configuration: configuration)
+let manager = NetworkManager(manager: sessionManager )
+```
+
+After this you are ready to go! You can also change the way you see the response (by default you will see it displayed in a window on top of the status bar).
```
Comment (patriciapoj2017) [inline _posts/2017-05-02-Introducing-to-Xniffer.md]: Now you're ready to go!

Pregunta: la response es una sola o son varias? Por lo que entendemos son varias y sería responses, pero capaz es solo una....si fueran más va la frase abajo. 

 You can also change the way you see the responses (by default you will see them displayed in a window on top of the status bar).

File: _posts/2017-05-02-Introducing-to-Xniffer.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+### UI Modes
+The Xniffer currently supports three modes, `.window`, `.console` and `.custom`. The default value is `.window` but this can be changed easily on the setup: `Xniffer.setup(with configuration: URLSessionConfiguration, mode: XnifferUI = .window)`. Each one of this uses a different implementation of the `XnifferDelegate`.
```
Comment (patriciapoj2017) [inline _posts/2017-05-02-Introducing-to-Xniffer.md]: no es necesario el "of this" queda mejor Each one uses a different implementation...

File: _posts/2017-05-02-Introducing-to-Xniffer.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+
+- `.window`: Displays a window on top of the status bar which can be expanded to display a list of the profiled requests. This is the default value.
+- `.console`: Prints the results on the Xcode console.
+- `.custom`: This one receives a closure of type `() -> ()` so you can use your own implementation of the `XnifferDelegate`.
+
+To see these three different ways of displaying results we invite you to visit our [Github project](https://github.com/xmartlabs/Xniffer).
```
Comment (patriciapoj2017) [inline _posts/2017-05-02-Introducing-to-Xniffer.md]: displaying the results

File: _posts/2017-05-02-Introducing-to-Xniffer.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+Carthage is a simple, decentralized dependency manager for Cocoa.
+
+To install Xniffer, simply add the following line to your Cartfile:
+```
+github "xmartlabs/Xniffer" ~> 1.0
+```
+
+
+## Where to go from here
+We hope you find this post as a good introduction to this open source library. On GitHub you will find everything you need to get started or if you want to collaborate feel free to [contribute](https://github.com/xmartlabs/Xniffer) with this library.
```
Comment (patriciapoj2017) [inline _posts/2017-05-02-Introducing-to-Xniffer.md]: We hope this works as a good introduction to this open source library. On GitHub you'll find everything you need to get started. If you want to collaborate, feel free to contribute with this library.

(sacamos el duplicado the this en la primera oración, agregamos contractions y frases cortas para lectura más ágil

**Pull request merged**